src/crimson/os/seastore/cached_extent.h

```diff
@@ -783,6 +790,12 @@ class CachedExtent
     prior_instance.reset();
   }
 
+  void update_checksum() {
+    auto crc = calc_crc32c();
+    set_last_committed_crc(crc);
+    update_in_extent_chksum_field(crc);
```

Comment:
Let's decouple `update_in_extent_chksum_field` out of `update_checksum`, see another comment.

And then `update_checksum()` worths to have a better name `calc_last_committed_crc()`.

Reply:
Edit: looks some other places reuses `update_checksum()` with `update_in_extent_chksum_field()` as well.

The above is only a suggestion.